app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
```

Comment:
- [ ] Kindly specify the `foreign_key` for the User model's associations here for the application to know that this specific model is what you're calling the `author`.
Ex:
`has_many :posts, foreign_key: 'author_id'`